nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -63,6 +63,9 @@ class FpgaPesEnsembleNetwork(nengo.Network):
         that is used to handle UDP communication between the host PC and the
         FPGA board. The full list of parameters can be found here:
         https://github.com/nengo/nengo-fpga/blob/master/nengo_fpga/sockets.py#L425
+    recurrence : float or (D_out, D_in) array_like, optional
```

Comment:
I wouldn't worry too much about being consistent with Nengo SPA, but personally I prefer `feedback` because you don't have to remember that it's "recurrence" rather than "recurrent" or some other form of the word. `feedback` is kind of the only option for that word.

Reply:
So are you proposing we change the constructor argument (`recurrence`) or the recurrent connection object (`self.recurrent`)? or both?

Reply:
I was only looking at the constructor argument, but I would imagine you'd want to be consistent internally unless there's some reason to be different.

Reply:
I would change both to `feedback`.